scenarios/kitsune_continuous_flow/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "kitsune_continuous_flow"
```

Comment:
This'll be scenario where messages are created in bursts of x messages every y seconds. Anyone offers a better name?

Reply:
Yeah, I'm thinking there'll be more kitsune scenarios and `op_sync` will apply to all of them in a way.

Reply:
Fair, in that case this seems good